app/javascript/components/listings/ListingApplicationsPage.js

```diff
@@ -31,9 +31,7 @@ const tableFields = {
     'label': 'Total Household Size'
   },
   'application_submission_type': {
-    'label': 'Application Submission Type',
-    'editable': true,
```

Comment:
IndexTable on it's own doesn't have an editing option for fields. We also don't want users to be able to change the application submission type in this table. 

See [SpreadsheetIndexTable](https://github.com/Exygy/sf-dahlia-lap/blob/master/app/javascript/components/SpreadsheetIndexTable.js#L37-L42) for an example of IndexTableCell edit-ability in action